src/workers/parser/getSymbols.js

```diff
@@ -122,6 +123,12 @@ function getFunctionParameterNames(path: SimplePath): string[] {
   return [];
 }
 
+function addIdentifier(symbols, newId) {
+  if (!symbols.identifiers.find(id => nodeHasSameLocation(id, newId))) {
```

Comment:
i'm a little concerned by the perf here, but i think it is okay because we don't store that many identifiers...

Reply:
Should we add it and then dedup at the end instead maybe? At the end of the day this is fine though.